core/templates/components/question-directives/questions-list/questions-list.directive.html

```diff
@@ -15,11 +15,12 @@ <h3>Difficulty</h3>
       <div class="oppia-mobile-collapsible-card-content" ng-if="$ctrl.difficultyCardIsShown">
         <div class="list-group">
           <h4>Select Difficulty</h4>
-          <div ng-repeat="skillWithDifficulty in $ctrl.linkedSkillsWithDifficulty track by $index" ng-click="$ctrl.changeLinkedSkillDifficulty()">
```

Comment:
Why did we remove the ng-click?

Reply:
`changeLinkedSkillDifficulty()` function was executing before the updation of the array.